libs/features/pages/home/src/lib/main/home/home-page.component.html

```diff
@@ -1,3 +1,4 @@
 <div class="home-page">
-  Elewa Education
-</div>
\ No newline at end of file
+  <elewa-website-elewa-info-card [cardInfo]="{title: 'Course design', description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Praesentium minima tempora magnam iusto.', icon: 'https://static.vecteezy.com/system/resources/previews/005/226/469/non_2x/rising-arrow-and-bar-graph-silhouette-icon-free-vector.jpg'}"> </elewa-website-elewa-info-card>
```

Comment:
Custom Mock data should be stored in a data module. If not specified in the issue, then there's no need to push this changes